docs/running-locally.md

```diff
@@ -2,12 +2,12 @@
 
 ## Pre-requisites
 
-* Golang
-* Yarn. `brew install yarn` 
-* Docker
+* [Go](https://golang.org/dl/) (The project currently uses version 1.13)
```

Comment:
Specify the project uses version 1.13. Using 1.14 can provide unexpected codegen and lint results